src/actions/pubSubHandlers.js

```diff
@@ -74,29 +70,31 @@ export const handleNewMessage = message => (dispatch, getState) => {
   const state = getState()
   const channels = channelSelector(state)
   const user = userSelector(state)
-
-  if (message.author.id === user.id || findIndex(channels, {id: message.author.id}) !== -1) {
+  if (
+    message.author.id === user.id ||
+    findIndex(channels, { id: message.author.id }) !== -1 ||
+    message.type
```

Comment:
@nikgraf yes, it will add a new message. But activities messages is a special message which we have for example in `Development activities` channel. They have generated automatically, and for them, we don't need to open_pm. As in this condition, we have an author (those messages don't have the author) and we don't have the author in channels array (again, we don't have the author), we just need to execute `addNewMessage` and return without creating open_pm. Victor mentioned that this behaviour loads server so much. In last month server received 111000 (as I remember, may be wrong) of wrong creating open_pm.

I insert `message.type` because only activity messages have some type. If we will talk in private, our messages will be without any `type` at all.

Reply:
Makes a lot of sense now after your explanation. Thanks for clarifying. I added it as a comment and made an explicit if condition that returns early.

Good to be merged from my perspective!